web/modules/custom/weather/src/Form/WeatherUserForm.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Drupal\weather\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\weather\Services\WeatherApi;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * {@inheritdoc}
+ */
```

Comment:
You create a new class, according to the standards, you need to describe the logic of this class, and not use {@inheritDoc}